### PR TITLE
Droth 2527 prohibitions generated by traffic signs created without time validations

### PR DESCRIPTION
**Pull request by ricardofmsimoes**
Fix the generator of Linear prohibitions through by traffic sign

### CODE REVIEW

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala [hunk 1]
```diff
@@ -233,9 +233,18 @@ class TrafficSignLinearGeneratorSpec extends FunSuite with Matchers {
...
   test("create prohibitions values based on trafficSigns with additional Panels") {
     val prohibitionGenerator = new TestTrafficSignProhibitionGenerator()
-    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "9-10","", 1, "", 99, 99, 99), AdditionalPanel(ValidSat.OTHvalue, "(11-12)","", 2, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "15-16 (17-18) 19-20","", 3, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "17-18","", 4, "", 99, 99, 99))
-    val prohibitionPeriod = Set(ValidityPeriod(9, 10, ValidityPeriodDayOfWeek.Weekday), ValidityPeriod(11, 12, ValidityPeriodDayOfWeek.Saturday), ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Sunday)
-      ,ValidityPeriod(15, 16, ValidityPeriodDayOfWeek.Weekday) ,ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Saturday) ,ValidityPeriod(19, 20, ValidityPeriodDayOfWeek.Sunday))
+    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "", "9-10", 1, "", 99, 99, 99),
+                            AdditionalPanel(ValidSat.OTHvalue, "", "(11-12)",2, "", 99, 99, 99),
+                            AdditionalPanel(ValidMultiplePeriod.OTHvalue, "", "15-16 (17-18) 19-20", 3, "", 99, 99, 99),
```
Comment (marquesfm) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala:238]: This example doesn't exist anymore.
The old sign who allows the 3 prohibitions at the same time doesn't exist on new sign law.

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala [hunk 2]
```diff
@@ -250,9 +259,18 @@ class TrafficSignLinearGeneratorSpec extends FunSuite with Matchers {
 
   test("insert values  on trafficSigns with additional Panels") {
     val prohibitionGenerator = new TestTrafficSignProhibitionGenerator()
-    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "9-10","", 1, "", 99, 99, 99), AdditionalPanel(ValidSat.OTHvalue, "(11-12)","", 2, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "15-16 (17-18) 19-20","", 3, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "17-18","", 4, "", 99, 99, 99))
-    val prohibitionPeriod = Set(ValidityPeriod(9, 10, ValidityPeriodDayOfWeek.Weekday), ValidityPeriod(11, 12, ValidityPeriodDayOfWeek.Saturday), ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Sunday)
-      ,ValidityPeriod(15, 16, ValidityPeriodDayOfWeek.Weekday) ,ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Saturday) ,ValidityPeriod(19, 20, ValidityPeriodDayOfWeek.Sunday))
+    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "", "9-10", 1, "", 99, 99, 99),
+                          AdditionalPanel(ValidSat.OTHvalue, "", "(11-12)", 2, "", 99, 99, 99),
+                          AdditionalPanel(ValidMultiplePeriod.OTHvalue, "", "15-16 (17-18) 19-20", 3, "", 99, 99, 99),
```
Comment (marquesfm) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala:264]: Same as mentioned above.

**Review by marquesfm — COMMENTED** [review]
OK with a few suggestions.

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala (hunk 1 above)
Comment (ricardofmsimoes) [reply digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala]: Done.

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala (hunk 2 above)
Comment (ricardofmsimoes) [reply digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala]: Done.

**Pull request merged**